### PR TITLE
Multi-Tenant RBAC template

### PR DESCRIPTION
**Pull request by ronenh**

### CODE REVIEW

File: assets/multi-tenant/ds-load/multi-tenant.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      },
+      {
+        "type": "resource",
+        "id": "adventures",
+        "display_name": "The Citadel Adventures resource",
+        "properties": {}
+      },
+      {
+        "type": "resource",
+        "id": "buget",
```
Comment (ogazitt) [inline assets/multi-tenant/ds-load/multi-tenant.json:23]: budget

File: assets/multi-tenant/ds-load/multi-tenant.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        "display_name": "The Smiths family's buget",
```
Comment (ogazitt) [inline assets/multi-tenant/ds-load/multi-tenant.json:24]: budget

File: assets/multi-tenant/ds-load/multi-tenant.json [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        {
+          "object_type": "tenant",
+          "object_id": "the-smiths",
+          "relation": "viewer",
+          "subject_type": "user",
+          "subject_id": "rick@the-citadel.com"
+        },
+        {
+          "object_type": "resource",
+          "object_id": "buget",
```
Comment (ogazitt) [inline assets/multi-tenant/ds-load/multi-tenant.json:80]: budget

File: assets/multi-tenant/manifest.yaml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+      can_edit: editor | can_administer | system->editor
+      can_view: viewer | can_edit | system->viewer
+
+      # fine-grained permissions make it easier to change access levels without
+      # modifying application logic.
+      can_delete_tenant: owner | system->admin
+      can_manage_roles: can_administer
+      can_list_members: can_view
+      # an owner cannot leave the tenant. they must be removed by another owner.
+      can_leave_tenant: can_view - owner
```
Comment (ogazitt) [inline assets/multi-tenant/manifest.yaml:61]: doesn't this imply another permission - `can_remove_member`?

**Review by ogazitt — CHANGES_REQUESTED** [review]

File: assets/multi-tenant/manifest.yaml (hunk 4 above)
Comment (ronenh) [reply assets/multi-tenant/manifest.yaml:61]: How about renaming `can_manage_roles` to `can_manage_members`? I meant for that permissions to cover adding/removing/updating tenant members.

**Review by ogazitt — DISMISSED** [review]

**Comment by ronenh** [comment]
This has already been merged

**Pull request closed without merging**